com.unity.uiwidgets/Runtime/ui2/text.cs

```diff
@@ -574,7 +574,15 @@ Locale locale
         internal static unsafe void setFloat(this byte[] bytes, int byteOffset, float value) {
             D.assert(byteOffset >= 0 && byteOffset + 4 < bytes.Length);
             fixed (byte* b = &bytes[byteOffset]) {
+#if UNITY_ANDROID && !UNITY_EDITOR
```

Comment:
to avoid allocation in every call, can we do something like this:

```
static byte[] _scalar_as_2s_compliment_vars = new byte[4];


            static unsafe int GetBytesToInt32(float value) {
                var intVal = *(int*) &value;
                fixed (byte* b = _scalar_as_2s_compliment_vars) {
                    *((int*) b) = intVal;
                }

                fixed (byte* pbyte = &_scalar_as_2s_compliment_vars[0]) {
                    return *((int*) pbyte);
                }
            }
```